### PR TITLE
Bug 1376349 - mydashboard.html \ "Updated Since Last Visit" \ "Mark Visited" should do also a "Refresh"

### PR DESCRIPTION
**Pull request by SebastinSanty**

### CODE REVIEW

File: extensions/MyDashboard/web/js/query.js [hunk 1]
```diff
@@ -271,6 +271,11 @@ $(function() {
             }
             YAHOO.bugzilla.bugUserLastVisit.update(bug_ids);
             YAHOO.bugzilla.bugInterest.unmark(bug_ids);
+
+	    var query_select = Y.one('#query');
```
Comment (dylanwh) [inline extensions/MyDashboard/web/js/query.js]: the indentation here should be consistent with the rest of the file (and not use tabs)
otherwise this looks good!

File: extensions/MyDashboard/template/en/default/pages/mydashboard.html.tmpl [hunk 2]
```diff
@@ -108,7 +108,7 @@
         <span id="query_count_refresh" class="bz_default_hidden">
           <span class="items_found" id="query_bugs_found">0 [% terms.bugs %] found</span>
           | <a class="refresh"     href="javascript:void(0);" id="query_refresh">Refresh</a>
-          <span id="bar_markread">|</span> <a class="markread" href="javascript:void(0);" id="query_markread">none</a>
+          <span id="bar_markread">|</span> <a class="refresh" href="javascript:void(0);" id="query_markread">none</a>
```
Comment (dylanwh) [inline extensions/MyDashboard/template/en/default/pages/mydashboard.html.tmpl]: Help me understand why this change is needed?

File: extensions/MyDashboard/web/js/query.js (hunk 1 above)
Comment (Dipanks) [inline extensions/MyDashboard/web/js/query.js]: I will change this and re commit.

File: extensions/MyDashboard/template/en/default/pages/mydashboard.html.tmpl (hunk 2 above)
Comment (Dipanks) [inline extensions/MyDashboard/template/en/default/pages/mydashboard.html.tmpl]: This change is absolutely not needed. Probably some mistake on my part while committing.

**Comment by Dipanks** [comment]
@dylanwh , Is this okay? I am not sure why I am unable to push properly for this bug. I pushed the other one lasst time.

**Comment by Dipanks** [comment]
@dylanwh ,shall I open a new pull request for this? This one is getting dirty.

**Comment by SebastinSanty** [comment]
@Dipanks Maybe because you rebased last time, so now you have to do a forced update.

**Comment by Dipanks** [comment]
@SebastinSanty should i do one more forced update? Dylan was helping me with this issue few minutes back.

**Comment by dylanwh** [comment]
This one is still not right. I'm not sure what the code is doing?

**Comment by Dipanks** [comment]
@dylan I took the code from query_refresh function and added it to the end of query_markread function. 
Once query_markread is selected, it will mark as read and then refresh the table. Do I need to change anything??
